sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -170,57 +175,56 @@ export type JsonWebKeyOperation = "encrypt" | "decrypt" | "sign" | "verify" | "w
 // @public
 export type JsonWebKeyType = "EC" | "EC-HSM" | "RSA" | "RSA-HSM" | "oct";
 
-// @public
-export interface Key {
-    keyMaterial?: JsonWebKey;
-    keyOperations?: JsonWebKeyOperation[];
-    keyType?: JsonWebKeyType;
-    properties: KeyProperties;
-}
-
 // @public
 export class KeyClient {
     constructor(endPoint: string, credential: TokenCredential, pipelineOrOptions?: ServiceClientOptions | NewPipelineOptions);
```

Comment:
Unrelated to this pr: endpoint is one word 😀

Reply:
I'll solve this in this PR